### PR TITLE
backlight: enable changing via scroll

### PR DESCRIPTION
**Pull request by devsnek**
TEAM EDIT:

Adds the `enable-scroll` key in the backlight module.

### CODE REVIEW

**Comment by patrick96** [comment]
Hi, thanks for helping out :)

I couldn't test this yet, but this tries to write to /sys/class/backlight/.../backlight, doesn't this require root?

**Comment by devsnek** [comment]
@patrick96 many systems are configured for the video group to have write on backlight (usually this coincides with people who use bars/i3/bspwm/etc, for exactly this reason)

File: src/modules/backlight.cpp [hunk 1]
```diff
@@ -88,6 +112,30 @@ namespace modules {
...
+    }
+
+    m_log.info("%s: Changing value by %f%", name(), value_mod);
+
+    try {
+      int rounded = math_util::cap<double>(m_percentage + value_mod, 0.0, 100.0) + 0.5;
+      int value = math_util::percentage_to_value<int>(rounded, m_max_brightness);
+      file_util::write_contents(m_path_backlight + "/brightness", to_string(value));
+    } catch (const exception& err) {
+      m_log.err("%s: %s", name(), err.what());
```
Comment (patrick96) [inline src/modules/backlight.cpp]:
```suggestion
      m_log.err(
          "%s: Couldn't change backlight value. You may need to configure your system, please read the module "
          "documentation.\n(reason: %s)",
          name(), err.what());
```

**Comment by devsnek** [comment]
ping

File: src/modules/backlight.cpp [hunk 2]
```diff
@@ -76,6 +80,25 @@ namespace modules {
...
+    // the format prefix/suffix also gets wrapped
+    // with the cmd handlers
+    string output{module::get_output()};
+
+    if (m_scroll) {
+      m_builder->cmd(mousebtn::SCROLL_UP, EVENT_SCROLLUP);
+      m_builder->cmd(mousebtn::SCROLL_DOWN, EVENT_SCROLLDOWN);
+    }
+
+    m_builder->append(output);
```
Comment (Lomadriel) [inline src/modules/backlight.cpp]:
```suggestion
    m_builder->append(std::move(output));
```

Just add a merge here and it'll be ready to be merged for me.

**Pull request merged**